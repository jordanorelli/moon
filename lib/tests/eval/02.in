# simple keys and values

key: 1 # comments are stripped
key_2: 2
key_3: -1

# basically the stripping of comments is all we care about
key_4: 0
key_5: +0

key_6: this is a string
key_7: "this is a quoted string"

key_8: [1 2 3]

