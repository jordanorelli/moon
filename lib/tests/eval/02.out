key: 1
key_2: 2
key_3: -1
key_4: 0
key_5: +0
key_6: this is a string
key_7: "this is a quoted string"
key_8: [1 2 3]

